src/dmd/inline.d

```diff
@@ -38,6 +38,7 @@ import dmd.init;
 import dmd.initsem;
 import dmd.mtype;
 import dmd.opover;
+import dmd.printast;
```

Comment:
What's this for?

Reply:
So I can call `printAST(e)` for debugging purposes. I keep adding them back in to the imports, and they keep getting deleted.

Reply:
Why does it need to remain after the debugging has finished? A good D file should not have ~25 lines of unspecific imports.

Also note that `imported` is in druntime as of a few releases ago. You can't use it in the compiler source yet but you can use it when debugging.